designs/forceful-expiration.md

```diff
@@ -0,0 +1,64 @@
+# Forceful Expiration - v0.37+
```

Comment:
Would calling Graceful "Launch before Terminate" and Forceful as "Terminate and launch" make more sense? Both flow is still graceful they just order the node creation differently

Reply:
I think there's value in having one-word terms to map to the phrase, but I do agree those are pretty clear. There's another mechanism in between called Eventual that's not called out here that is harder to explain in those terms as well.